webrtc.html

```diff
@@ -4274,6 +4274,15 @@ <h2>Attributes</h2>
                   itself, browsers can offer their users different policies
                   regarding sharing local addresses, as defined in
                   [[RTCWEB-IP-HANDLING]].</p>
+                  <p>Remote candidates may also be exposed, for instance
```

Comment:
You need to end the <div class=note> starting at line 4253 before this, since the new text is intended to be normative.

Reply:
Fixed